parlai/agents/torch/torch.py

```diff
@@ -0,0 +1,257 @@
+# Copyright (c) 2017-present, Facebook, Inc.
```

Comment:
should it be in core?

Reply:
I wasn't sure as it's specific, but I could see it living in core because it's task-independent. I'm leaning towards moving it to core.